odoo/tools/__init__.py

```diff
@@ -12,6 +12,12 @@
 from .i18n import format_list, py_to_js_locale
 from .image import image_process
 from .json import json_default
+
+# Monkeypatch -- werkzeug_urls requires json_default
+# and is needed by modules below, i.e. `mail.validate_url`
+from odoo._monkeypatches.werkzeug_urls import patch_werkzeug_urls
```

Comment:
Not needed as we patch dynamically what is needed (patching is done when starting odoo. If we want to make sure that this module could be loaded before the patching, it should just import the package and then resolve the necessary functions instead of using `from ... import somefunc`.